src/curobo/curobolib/cpp/self_collision_kernel.cu

```diff
@@ -175,8 +175,6 @@ namespace Curobo
       {
         sph2 = __rs_shared[j]; // we need not load sph2 in every iteration.
 
-#pragma unroll 16
```

Comment:
I don't believe ndpt is known at compile time?

Reply:
ndpt is not known. You are correct.